misc/pytorch_toolkit/noise_suppression/states.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+def get_shape(x):
+    return list(x.shape)
+    #this is to get more compact and human readable but unresizable onnx model
+    #return [int(s) for s in x.shape]
```

Comment:
This could be imported from the `model.py` or moved to some common file like `utils.py`

Reply:
moved to utils